File: Dinner 18_10_2025/leapYear.js
const prompt = require("prompt-sync")({signit: true});
 
let userInput = prompt("Enter a number: ")
if ( userInput % 4 == 0 && userInput % 100 != 0){
	console.log("Is a leap year");
}
else{
	console.log("Not a leap year");
}

File: Dinner 18_10_2025/leapYear.java
import java.util.Scanner;

public class leapYear{
	public static void main(String[] args){

		Scanner input = new Scanner(System.in);
		System.out.print("Enter a year: ");
		int year = input.nextInt();
		
		if ( year % 4 == 0 && year % 100 != 0){
			System.out.print("Is a leap year");
		}
		else{
			System.out.print("Not a leap year");
		}
	}
}